<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripRequest</class>
 <widget class="QWidget" name="TripRequest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCommandLinkButton" name="BackBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>enter the destinations :</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="Destination">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>e.g : Galaxy1,Planet1  Galaxy2,Planet2  ...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'green'</string>
   </property>
   <property name="text">
    <string>Cost :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cost">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'green'</string>
   </property>
   <property name="text">
    <string>Distance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Distance">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>150</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-4</y>
     <width>451</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>BackBtn</zorder>
  <zorder>label</zorder>
  <zorder>Destination</zorder>
  <zorder>OkBtn</zorder>
  <zorder>label_2</zorder>
  <zorder>Cost</zorder>
  <zorder>label_3</zorder>
  <zorder>Distance</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
